src/dbnode/storage/database.go

```diff
@@ -992,7 +991,7 @@ func (d *db) WideQuery(
 	queryStart time.Time,
 	shards []uint32,
 	iterOpts index.IterationOptions,
-) ([]xio.IndexChecksum, error) { // FIXME: change when exact type known.
+) ([]xio.WideEntry, error) { // FIXME: change when exact type known.
```

Comment:
Should this `FIXME` still be here?

Reply:
Yeah, the upcoming PR has the iterator itself